api/pyproject.toml

```diff
@@ -60,7 +60,26 @@ test = [
   "pook @ git+https://github.com/h2non/pook.git@master",
   "pytest >=7.4.4, <8",
   "pytest-django >=4.8.0, <5",
+  "pytest-pook>=1.0.0",
   "pytest-raises >=0.11, <0.12",
   "pytest-sugar >=0.9.7, <0.10",
   "schemathesis >=3.25.6, <4",
 ]
+
+[tool.pytest.ini_options]
+DJANGO_SETTINGS_MODULE = "conf.settings"
+
+pythonpath = "."
+
+filterwarnings = [
+  # Ignore warnings related to unverified HTTPS requests.
+  # Reason: This warning is suppressed to avoid raising warnings when making HTTP requests
+  # to servers with invalid or self-signed SSL certificates. It allows the tests to proceed
+  # without being interrupted by these warnings.
```

Comment:
Great docs :+1:

Reply:
To clarify, I copied these from the pytest.ini, didn't write them myself!